tinker/static/assets/css/tinker.css

```diff
@@ -560,7 +560,26 @@ label {
 }
 
 .table > thead > tr > th, .table > tbody > tr > th, .table > tfoot > tr > th, .table > thead > tr > td, .table > tbody > tr > td, .table > tfoot > tr > td {
-    vertical-align: unset !important;
+    /*vertical-align: unset !important;*/
```

Comment:
If this isn't used anymore, it might be worth deleting?

Although, this is probably used for other tables. We should check the other tables aren't broken.

Reply:
Had to keep in for other tables, made the css for the redirect-table specific to that table.